Major changes in pl.js

ectryliz opened the pull request.

ectryliz commented:
Missed some stuff, going to update them tomorrow.

xhg4 commented:
First of all, thanks for the massive contribution!
I will review the changes on Sunday (or at least that's the plan)
From what I've seen so far tho, the changes are generally good ones but I'll have to carefully look through them

ectryliz commented:
For your information: I have fixed the consistency between translations, fixed all of the typos and punctuation errors. Changed what seem to be literal or incomprehensible translations and some small stylistic changes.

xhg4 commented on pl.js:
```diff
@@ -812,7 +812,7 @@ social.market.fee.gift Brak podatku
 social.market.gift.msg Zostaw wiadomość
 respawn.auto Automatyczne odrodzenie za {0}
-custom.host.ranked Nie możesz hostować gier, gdy grasz w trybie rankowanym.
+custom.host.ranked Nie możesz hostować gier, gdy grasz w trybie rankingowym.
```
For the sake of consistency, I think "Nie możesz tworzyć rozgrywek..." or "Nie możesz utworzyć rozgrywki..." or maybe "Nie można tworzyć rozgrywek, grając w trybie rankingowym"

ectryliz replied:
Missed this one

xhg4 reviewed (commented):
Okay, I found 2 lines that could be changed but it's not really necessary
No idea how this many typos and even mistakes (like the one line were in the previous version only half of the phrase was translated) went unnoticed

xhg4 commented:
Btw, would you be willing to continue contributing to the Polish translation? If so, would you prefer to mostly do the translating or reviewing?

ectryliz commented:
Can do both, depends how much free time I have.

ectryliz commented:
@xhg4 There are some translations, that have word "rozgrywka". I think it would be better, if it got replaced with "gra". If you think the changes are wrong or seem extraordinary, I will revert them back.

xhg4 commented:
> @xhg4 There are some translations, that have word "rozgrywka". I think it would be better, if it got replaced with "gra". If you think the changes are wrong or seem extraordinary, I will revert them back.

Yeah, now that you mention it, I agree

If those are all the changes then it's ready to be merged @mty22

ectryliz commented:
Everything here is probably done, it's ready to be merged.

mty22 commented:
@ectryliz can you please reach out to me with your IGN?

mty22 commented:
nevermind, got it.
